lnwire/node_announcement.go

```diff
@@ -157,3 +157,54 @@ func (a *NodeAnnouncement) DataToSign() ([]byte, error) {
 
 	return w.Bytes(), nil
 }
+
+// IsEqual compares the configurable fields within two NodeAnnouncement
+// objects, and returns whether they are equal or not. The signature is not
+// compared.
```

Comment:
The latupdate time should be one of the compared fields?

Reply:
Not sure if it's really necessary as the timestamp's only purpose is to order messages.